.github/workflows/on-push.yml

```diff
@@ -29,18 +29,17 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
       
     #- name: Codespell with annotations
     #  uses: codespell-project/actions-codespell@v1.0
 
     
     - name: Install Conda Environment
-      uses: mamba-org/provision-with-micromamba@v13
+      uses: mamba-org/setup-micromamba@v1
       with:
         environment-file: /binder/environment.yml
-        cache-env: true
-        cache-env-key: ubuntu-latest-3.10
+        cache-environment: true
```

Comment:
Running into a build error, seems to be related to https://github.com/mamba-org/setup-micromamba/issues/130